firmware/quintuna/BMS/src/tasks.c

```diff
@@ -34,8 +35,12 @@ void tasks_init(void)
     hw_usb_init();
     hw_adcs_chipsInit();
     hw_pwms_init();
+    hw_can_init(&can1);
+    hw_can_init(&can2);
```

Comment:
isn't this the one where we want to init based on state machine? charging vs not charging?